modules/test_user_dashboard/app/services/test_user_dashboard/big_query.rb

```diff
@@ -7,21 +7,12 @@ module TestUserDashboard
   class BigQuery
     include SentryLogging
 
+    ENV['BIGQUERY_CREDENTIALS'] = Settings.test_user_dashboard.bigquery.to_json
```

Comment:
This might be better suited in an initializer OR if you want to take it a step further, on the container in the devops repo, but I think an initializer might make the most sense here.

Reply:
I move it to a new file - `config/initializers/test_user_dashboard.rb`